### ISSUE TITLE
Questions about training ACGAN 

### ISSUE THREAD

**Issue by zzw1123** [hwalsuklee/tensorflow-generative-model-collections#29]
Thanks for your contribution!
A problem occurs when I am training ACGAN without tuning any parameters which is shown as follows:
As the training process goes on, the loss of D is declining, but the loss of G is increasing on the contrary...and the gap between two of them is pretty large. I do not know how to solve it...
![image](https://user-images.githubusercontent.com/16879220/42943275-da3dc8d8-8b94-11e8-8a48-b0cd9e8c6893.png)
